src/dependency-manager/values/values.ts

```diff
@@ -47,7 +47,7 @@ export class ValuesConfig {
       if (typeof component_values === 'object' && !(component_values instanceof Array)) {
         for (const [param_key, param_value] of Object.entries(component_values || {})) {
```

Comment:
Do we want to update param_key and _value to secret?

Reply:
Similar for other instances of param_key mentioned in this file

Reply:
Good catch, I'll update